src/routes/routeTree.autogenerated.ts

```diff
@@ -0,0 +1,261 @@
+/* prettier-ignore-start */
+
+/* eslint-disable */
+
+// @ts-nocheck
+
+// noinspection JSUnusedGlobalSymbols
+
+// This file is auto-generated by TanStack Router
+
+import { createFileRoute } from '@tanstack/react-router';
+
+// Import Routes
+import { Route as rootRoute } from './__root';
+import { Route as LoginImport } from './login';
+import { Route as IndexImport } from './index';
+import { Route as UserUserImport } from './user/_user';
+import { Route as AdminAdminImport } from './admin/_admin';
+import { Route as UserUserXpubImport } from './user/_user.xpub';
+import { Route as UserUserDestinationsImport } from './user/_user.destinations';
+import { Route as UserUserAccessKeysImport } from './user/_user.access-keys';
+import { Route as AdminAdminXpubImport } from './admin/_admin.xpub';
+import { Route as AdminAdminTransactionsImport } from './admin/_admin.transactions';
+import { Route as AdminAdminPaymailsImport } from './admin/_admin.paymails';
+import { Route as AdminAdminDestinationsImport } from './admin/_admin.destinations';
+import { Route as AdminAdminContactsImport } from './admin/_admin.contacts';
+import { Route as AdminAdminAccessKeysImport } from './admin/_admin.access-keys';
+
+// Create Virtual Routes
+
+const UserImport = createFileRoute('/user')()
+const AdminImport = createFileRoute('/admin')()
+const AboutLazyImport = createFileRoute('/about')()
+
+// Create/Update Routes
+
+const UserRoute = UserImport.update({
+    path: '/user',
+    getParentRoute: () => rootRoute,
+} as any)
+
+const AdminRoute = AdminImport.update({
+    path: '/admin',
+    getParentRoute: () => rootRoute,
+} as any)
```

Comment:
Do we need to add `"as any"` if we have `"// @ts-nocheck"` at the top of the file 🤔 ?

Reply:
Ahh ok, it's autogenerated 🙈.
So I don't think we should worry about it.

Reply:
Yes. Also i want to do small research, because i have this feeling that this could be added in gitignore. 
Because i believe it's only needed in output/built files.